salesforce-composite/2.18/modules/ROOT/pages/salesforce-composite-connector-studio.adoc

```diff
@@ -111,7 +111,7 @@ The corresponding password
 The corresponding security token
 +
 .The basic authentication includes a consumer key, consumer secret, username, and password.
-image::salesforce-composite-studio-basic-auth.png[Studio basic auth]
+image::salesforce-composite-studio-basic-auth.png["Global configuration window for basic auth"]
```

Comment:
If this is referring to the same as in the below image, I'd use "Basic Authentication".